### PR TITLE
faq page content added along with tests

### LINKED ISSUE #122: FAQ Page
As a user, if I have questions, I expect the FAQ Page to give me clear answers.

AC:
- Create FAQ accordions.
- Should be able to view even if not logged in.
- Copy taken from old code.

### PR DESCRIPTION
**Pull request by claireolmstead**
closes #122 

https://github.com/LibertyDSNP/provider-dashboard/assets/43625033/13815abe-671c-4ab6-86be-c8eaf0386993

### CODE REVIEW

**Review by rlaferla — COMMENTED** [review]
Some Svelte components for the Q&A would improve the readability and maintainability of this.

File: src/components/AddControlKey.svelte [hunk 1]
```diff
@@ -103,7 +103,7 @@
     <li>Select the new control key from the dropdown list below.</li>
     <li>Click 'Add It.'</li>
     <li>This requires 3 signatures: two for the authorization payload, and one to send the transaction.</li>
-    <ul class="ml-6 list-disc">
+    <ul class="unordered-list">
```
Comment (rlaferla) [inline src/components/AddControlKey.svelte:106]: This seems unrelated to the FAQ PR.

File: src/components/BlockSection.svelte [hunk 2]
```diff
@@ -2,7 +2,7 @@
   export let title: string;
 </script>
 
-<div class="content-block flex flex-col gap-4">
+<div class="content-block column">
```
Comment (rlaferla) [inline src/components/BlockSection.svelte:5]: This seems to break the styling for other components like Provider Login.  I added additional styles and updated the class on ProviderLogin but it looks slightly different than the Figma.

File: src/routes/faq/+page.svelte [hunk 3]
```diff
@@ -1 +1,49 @@
...
+    {
+      q: 'How do I transact on Frequency as a provider?',
+      a: TransactAsProvider,
+    },
+  ];
+</script>
+
+<div class="content-block w-single-block text-left text-base">
+  <h2 class="section-title">FAQ's</h2>
+  {#each questions as item, index}
```
Comment (rlaferla) [inline src/routes/faq/+page.svelte:22]: To clarify my comment in the previous review, I think this should be it's own component:

e.g.  something similar to this:
 ```
<FAQ>
   <FAQEntry q="What is this?" a="It is this." />
   <FAQEntry q="What is that?" a="That's another question." />
 </FAQ>
```

**Review by rlaferla — CHANGES_REQUESTED** [review]

File: src/components/AddControlKey.svelte (hunk 1 above)
Comment (claireolmstead) [reply src/components/AddControlKey.svelte:106]: We are using the same styling in this file as we are in the answers for FAQ. Since the styling in the file above is now used in multiple paces, I created a class in app.css and applied the reusable class here.

**Review by rlaferla — APPROVED** [review]
Looks good!
